mni_atlas_reader/atlas_reader.py

```diff
@@ -379,12 +389,30 @@ def create_output(filename, atlas, voxelThresh=2, clusterExtend=5,
         analysis
     probabilityThreshold : int
         Probability threshold for when using a probabilistic atlas
+    outDir : str or None
+        The full or relative path of an output directory created files are
```

Comment:
What do you mean with full or relative? How does it know which one to choose?
Would it make more sense to say something like "Path to the output directory. If None is specified, output will be stored in the same folder as the input files."

Reply:
Either should work, as per the argparse help for `filename`. But I agree, I like your suggestion better

Reply:
Ah, ok. I don't have much `argparse` experience. If you want to change the description text. could you change that in your branch and send a new commit?

Reply:
Done